src/blockrelay/graphene_set.cpp

```diff
@@ -86,12 +86,12 @@ CGrapheneSet::CGrapheneSet(size_t _nReceiverUniverseItems,
 
         mapCheapHashes.clear();
 
-        std::vector<uint64_t> cheapHashes;
+        std::vector<uint64_t> cheapHashes(_itemHashes.size());
```

Comment:
Was `clang` warning about this? Seems like some kind of optimizer warning? Just curious, have not seen that as a warning yet. (Also goes for the one below)

Reply:
ran a static code analyzer and this was a complaint

Reply:
What analyzer? clang sa?